lib/orbit/lib/objects.js

```diff
@@ -275,4 +275,33 @@ var isNone = function(obj) {
   return obj === undefined || obj === null;
 };
 
-export { Class, clone, defineClass, expose, extend, extendClass, isArray, isObject, isNone };
+/**
+ Combines two objects values
+
+ @method merge
+ @for Orbit
+ @param {Object} base
+ @param {Object} source
+ @returns {Object}
+ */
+var merge =  function(base, source) {
+  var merged = clone(base);
+  if (source) {
+    Object.keys(source).forEach(function(field) {
+      if (source.hasOwnProperty(field)) {
+        var fieldDef = source[field];
+        if (fieldDef) {
+          merged[field] = fieldDef;
+        } else {
+          // fields defined as falsey should be removed
+          delete merged[field];
```

Comment:
I can understand checking for `undefined` and deleting those fields, but deleting any field with a falsey value is unexpected.